src/Core/ImageFunction/CMakeLists.txt

```diff
@@ -4,8 +4,9 @@ compare_to_baseline(EXAMPLE_NAME MultiplyKernelWithAnImageAtLocation
   BASELINE_PREFIX OutputBaseline
   )
 
-add_example(GaussianBlurImageFunction)
-compare_to_baseline(EXAMPLE_NAME GaussianBlurImageFunction
+# TODO: Evaluate if GaussianBlurImageFunctionEx duplicates the Examples/Filtering/CMakeLists.txt:GaussianBlurImageFunction
```

Comment:
https://github.com/InsightSoftwareConsortium/ITK/blob/master/Examples/Filtering/GaussianBlurImageFunction.cxx and https://github.com/InsightSoftwareConsortium/ITKExamples/blob/master/src/Core/ImageFunction/GaussianBlurImageFunction/Code.cxx seem to be duplicates. We should probably remove one of them.